spec/std/io/memory_spec.cr

```diff
@@ -226,6 +226,16 @@ describe IO::Memory do
     io.gets(1).should eq("d")
   end
 
+  it "can seek and truncate" do
+    io = IO::Memory.new "hello world"
+    io.seek(-6, Seek::Current)
```

Comment:
CI fails because you need to use `IO::Seek::Current` here.

Reply:
Already fixed just waiting for specs to finish running. There are actually a few mistakes.